CHANGES.md

```diff
@@ -73,6 +73,7 @@ Fixed:
   could lead to incorrectly randomized playlists (#2507, #2500).
 - Fixed srt output in listener mode to allow more than one listener at a time and
   prevent listening socket from being re-created on listener disconnection (#2556)
+- Fixed `metadata.json.parse` always returns an empty list (#2815).
```

Comment:
```suggestion
```
By convention, bugs fixing like this that will be released on a previous version are not reported again in the following changelog.